contracts/EpochTokenLocker.sol

```diff
@@ -46,37 +41,50 @@ contract EpochTokenLocker {
         uint32 stateIndex;
     }
 
-    function deposit(address token, uint amount) public {
+    function deposit(address token, uint256 amount) public {
         updateDepositsBalance(msg.sender, token);
         require(
             ERC20(token).transferFrom(msg.sender, address(this), amount),
             "Tokentransfer for deposit was not successful"
         );
-        balanceStates[msg.sender][token].pendingDeposits.amount = balanceStates[msg.sender][token].pendingDeposits.amount
+        balanceStates[msg.sender][token]
+            .pendingDeposits
+            .amount = balanceStates[msg.sender][token]
+            .pendingDeposits
+            .amount
             .add(amount);
-        balanceStates[msg.sender][token].pendingDeposits.stateIndex = getCurrentBatchId();
+        balanceStates[msg.sender][token]
+            .pendingDeposits
+            .stateIndex = getCurrentBatchId();
         emit Deposit(msg.sender, token, amount, getCurrentBatchId());
     }
 
-    function requestWithdraw(address token, uint amount) public {
-        balanceStates[msg.sender][token].pendingWithdraws = PendingFlux({ amount: amount, stateIndex: getCurrentBatchId() });
+    function requestWithdraw(address token, uint256 amount) public {
+        balanceStates[msg.sender][token].pendingWithdraws = PendingFlux({
+            amount: amount,
+            stateIndex: getCurrentBatchId()
+        });
         emit WithdrawRequest(msg.sender, token, amount, getCurrentBatchId());
     }
 
     function withdraw(address token) public {
         updateDepositsBalance(msg.sender, token); // withdrawn amount might just be deposited before
 
         require(
-            balanceStates[msg.sender][token].pendingWithdraws.stateIndex < getCurrentBatchId(),
+            balanceStates[msg.sender][token].pendingWithdraws.stateIndex <
+                getCurrentBatchId(),
             "withdraw was not registered previously"
         );
 
-        uint amount = Math.min(
+        uint256 amount = Math.min(
             balanceStates[msg.sender][token].balance,
             balanceStates[msg.sender][token].pendingWithdraws.amount
         );
 
-        balanceStates[msg.sender][token].balance = balanceStates[msg.sender][token].balance.sub(amount);
+        balanceStates[msg.sender][token].balance = balanceStates[msg
```

Comment:
Here, the tool is joking us!

Reply:
Should probably be an issue in their github?

Reply:
I opened an issue over here:
https://github.com/prettier-solidity/prettier-plugin-solidity/issues/205